step-templates/ansible-run-playbook.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Id": "f0af011b-f99d-4809-84fe-af005b619fa2",
+  "Name": "Run Ansible Playbook",
+  "Description": "Step template to run an ansible playbook using bash",
+  "ActionType": "Octopus.Script",
+  "Version": 1,
+  "CommunityActionTemplateId": null,
+  "Packages": [],
+  "Properties": {
+    "Octopus.Action.Script.ScriptSource": "Inline",
+    "Octopus.Action.Script.Syntax": "Bash",
+    "Octopus.Action.Script.ScriptBody": "ansible-playbook $(get_octopusvariable \"Ansible.Playbook.Path\")\nplaybookRC=$?\n\nif [ $playbookRC -ne 0 ]; then\n  exit $playbookRC;\nfi"
```

Comment:
It looks like this calls an external program `ansible-playbook` ? Should the script check if its available, and error first with a helpful error message? (Just a suggestion)